lib/collection/src/shards/local_shard/scroll.rs

```diff
@@ -338,7 +367,7 @@ impl LocalShard {
             ))
         })?;
 
-        let mut rng = rand::thread_rng();
+        let mut rng = StdRng::from_entropy();
```

Comment:
What's the reason for switching from `rand::thread_rng` to `StdRng::from_entropy` here?

I'd expect the previous one to be faster because of reusing seeded material (though I haven't tested this).

Reply:
Thanks for pointing this, I forgot to add a comment.

The `rand::thread_rng` is not `Send` and got me into compilation issues.

Reply:
Makes sense, thanks!

Reply:
I think we can also just put `rand::thread_rng()` in the place we use `rng`

Reply:
So you mean calling `rand::thread_rng()` at every loop iteration?

Do you believe it will be faster than a single `StdRng::from_entropy` because of memoization or something?

No strong opinion on my side :)

Reply:
Based on a quick skim I thought the number of iterations would be very small, but that may not be the case.

Let's keep it like this. Sorry for the noise.

Reply:
No worries, it is always better to be thorough!